### PR TITLE
Add chocolate.bib

### PR DESCRIPTION
**Pull request by ThiloteE**
Library that can be used as (example) material for the JabRef website and other public relations matters.

The particular topic "chocolate" was chosen following a suggestion by Tobias Diez:

"For jabref online, I spend some time thinking about what kind of articles / subjects one should choose for user-faced presentation. In the end, I went for the theme "Chocolate" because:
1. It's positively received across many cultures
2. It's easy to understand and relate too, even if you are not an expert in the field
3. It has enough variation to e.g. demonstrate the group tree
4. while still being homogenous enough to at least make the impression to be a plausible "real" library of someone
5. It's not such a dry subject as computer science or mathematics, while at the same time being serious and scientific enough

Was hard to find a topic that scored well on all these points. Another close contender was "Old-time classics" (Newton, Einstein, Hilbert, ...), but then you get the cliché old western guys narrative. So maybe the blog series could use "Chocolate" as well?"

### CODE REVIEW

File: src/test/resources/testbib/Chocolate.bib [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+  eprint       = {https://jamanetwork.com/journals/jama/articlepdf/350738/jama\_210\_11\_011.pdf},
+  issn         = {0098-7484},
+  number       = {11},
+  pages        = {2071-2074},
+  url          = {https://doi.org/10.1001/jama.1969.03160370055011},
+  volume       = {210},
+  abstract     = {{To test the widespread idea that chocolate is harmful in instances of acne vulgaris, 65 subjects with moderate acne ate either a bar containing ten times the amount of chocolate in a typical bar, or an identical-appearing bar which contained no chocolate. Counting of all the lesions on one side of the face before and after each ingestion period indicated no difference between the bars. Five normal subjects ingested two enriched chocolate bars daily for one month; this represented a daily addition of the diet of 1,200 calories, of which about half was vegetable fat. This excessive intake of chocolate and fat did not alter the composition or output of sebum. A review of studies purporting to show that diets high in carbohydrate or fat stimulate sebaceous secretion and adversely affect acne vulgaris indicates that these claims are unproved.}},
```
Comment (Siedlerchr) [inline src/test/resources/testbib/Chocolate.bib:210]: is that paper open access? because abstracts tend to be copyrighted

**Comment by ThiloteE** [comment]
They are not all open access. I copied the bibtex data from scholar.google.de, then imported into JabRef, then with JabRef looked up the DOI, then imported data from DOI. There were two entries left without DOI, so I looked them up and imported the bibtex data from the journals website, which in this case included the abstract.

File: src/test/resources/testbib/Chocolate.bib (hunk 1 above)
Comment (ThiloteE) [reply src/test/resources/testbib/Chocolate.bib:210]: They are not all open access. I copied the bibtex data from scholar.google.de, then imported into JabRef, then with JabRef looked up the DOI, then imported data from DOI. There were two entries left without DOI, so I looked them up and imported the bibtex data from the journals website, which in this case included the abstract.

Comment (Siedlerchr) [reply src/test/resources/testbib/Chocolate.bib:210]: fine, then just remove the abstract

**Pull request merged**